src/gui/advancedsettings.h

```diff
@@ -58,7 +58,7 @@ private slots:
     template <typename T> void addRow(int row, const QString &text, T *widget);
 
     QSpinBox m_spinBoxAsyncIOThreads, m_spinBoxFilePoolSize, m_spinBoxCheckingMemUsage, m_spinBoxCache,
-             m_spinBoxSaveResumeDataInterval, m_spinBoxOutgoingPortsMin, m_spinBoxOutgoingPortsMax, m_spinBoxListRefresh,
+             m_spinBoxSaveResumeDataInterval, m_spinBoxOutgoingPortsMin, m_spinBoxOutgoingPortsMax, m_spinBoxStopTrackerTimeout, m_spinBoxListRefresh,
```

Comment:
Move `m_spinBoxStopTrackerTimeout` between `m_spinBoxAsyncIOThreads` and `m_spinBoxFilePoolSize`, to be consistent with the order everywhere.